Sources/Logto/Core/LogtoCore+Fetch.swift

```diff
@@ -93,25 +98,26 @@ extension LogtoCore {
         scopes: [String] = [],
         completion: @escaping HttpCompletion<CodeTokenResponse>
     ) {
-        let body: [String: Any] = [
+        let body: [String: String?] = [
             "grant_type": TokenGrantType.refreshToken.rawValue,
             "refresh_token": refreshToken,
             "client_id": clientId,
-            "resource": resource as Any,
+            "resource[]": resource,
```

Comment:
Noob question: what does this mean in swift? Converting string to a string array?

Reply:
just the convention from `oidc-provider` for parameter `resource`

Reply:
since it's designed to have the ability to accept multiple values